astropy/coordinates/solar_system.py

```diff
@@ -164,13 +165,12 @@ def _get_kernel(value):
     if value is None or value.lower() == "builtin":
         return None
 
-    try:
-        from jplephem.spk import SPK
-    except ImportError:
-        raise ImportError(
+    if not HAS_JPLEPHEM:
+        raise ModuleNotFoundError(
```

Comment:
Can we maybe keep the error type the same?

Reply:
[`ModuleNotFoundError`](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) is a subclass of [`ImportError`](https://docs.python.org/3/library/exceptions.html#ImportError), so this change is backwards compatible.